xcode/Subconscious/Shared/Components/Common/Recovery/RecoveryView.swift

```diff
@@ -37,54 +37,28 @@ struct AttemptRecoveryLabel: View {
     }
 }
 
-enum RecoveryViewTab {
-    case explain
+enum RecoveryViewStep: Hashable {
     case form
 }
 
 struct RecoveryView: View {
-    @ObservedObject var app: Store<AppModel>
-    var store: ViewStore<RecoveryModeModel> {
-        app.viewStore(get: RecoveryModeCursor.get, tag: RecoveryModeCursor.tag)
-    }
```

Comment:
We should pass ViewStore down from parent context. Creating it in a getter like this has two drawbacks:

- Child view must be exposed to full parent store, entangling child view in parent concerns.
- Getter creates a new viewStore every time you access it